tests/Hedgehog.Tests/Main.fs

```diff
@@ -0,0 +1,6 @@
+module test_expecto
+open Expecto
+
+[<EntryPoint>]
+let main argv =
+    Tests.runTestsInAssembly defaultConfig argv
```

Comment:
Do we want this file or `tests/Hedgehog.Tests/Program.fs`?

Reply:
Ooops. `Main.fs` shouldn't be commited. It is a leftover from trying `dotnet test` to work.
Removed it.